app/config/custom_tags.yml

```diff
@@ -0,0 +1,87 @@
+ezpublish:
+    system:
+        admin_group:
```

Comment:
Is that correct ? Or is the frontend also part of the `admin_group` ?
I expected that a tag needs to be enabled for both frontend and backend.

Reply:
> I expected that a tag needs to be enabled for both frontend and backend.

Actually... no. It's about being able to add tag in Online Editor. What was saved has to be rendered on frontend. Let's imagine a case where someone decides to no longer support some tag. In Online Editor it will result in validation error on save, which is IMHO fine. However if we added such checking on frontend it would also make sense to validate it. Such thing would result in fronted site crashing. 
So I'm not a huge fan of the idea.

But... We could introduce that for frontend by saying "let's just silently not render it if it's not enabled". The problem I have here is Product stability. This change would be a BC break after `2.1`. We still have some time until stable 2.1, but I think it's too little to do it.

Reply:
I see. Let's not break anything :)

The priority is to make sure the Admin won't break if there's a custom tag, whatever the reason, in a richtext field (after a migration for instance). It could throw a warning though.

Related to that, what about validation of custom tag attributes on publishing ? We didn't have that feature before. Isn't there a possibility that content that was valid before gets flagged as invalid ? If it is the case, we may want to make validation optional, disable it by default, and notify that having it to false is deprecated, and the default will be changed in the next major version.

We can think of the same procedure as for enabled/disabled tags in the context of rendering.

Reply:
You've touched interesting subject, let's talk about it internally.